docs/user/tutorials/01-20-22-gcp-scheduled-nfs-backup.md

```diff
@@ -35,23 +36,33 @@ This tutorial explains how to create scheduled automatic backups for ReadWriteMa
    ```
 
 3. Wait for the GcpNfsVolumeBackup to be in the `Active` state.
+
    ```shell
    kubectl -n $NAMESPACE wait --for=jsonpath='{.status.state}'=Active gcpnfsbackupschedule/my-backup-schedule --timeout=300s
    ```
+
    Once the GcpNfsVolumeBackup is created, you should see the following message:
-   ```
+
+   ```console
    gcpnfsbackupschedule.cloud-resources.kyma-project.io/my-backup-schedule condition met
    ```
+
 4. Observe the nextRunTimes for creating the backups.
+
    ```shell
    kubectl -n $NAMESPACE get gcpnfsbackupschedule my-backup-schedule -o jsonpath='{.status.nextRunTimes}{"\n"}' 
    ```
+
 5. Wait till the time specified in the nextRunTimes (in the previous step) and see that the GcpNfsVolumeBackup objects get created.
+
    ```shell
    kubectl -n $NAMESPACE get gcpnfsvolumebackup -l cloud-resources.kyma-project.io/scheduleName=my-backup-schedule 
    ```
-6. Clean up:
-    * Remove the created schedule and the backups:
-      ```shell
-      kubectl delete -n $NAMESPACE gcpnfsbackupschedule my-backup-schedule
-      ```
\ No newline at end of file
+
+## Next Steps
+
+To clean up, remove the created schedule and the backups:
+
+   ```shell
+   kubectl delete -n $NAMESPACE gcpnfsbackupschedule my-backup-schedule
+   ``
```

Comment:
```suggestion
   ```shell
   kubectl delete -n $NAMESPACE gcpnfsbackupschedule my-backup-schedule
   ```
```

Reply:
One backtick needs to be added in line 68. I added it in the suggestion above, but somehow, it doesn't appear there.
![Screenshot 2024-11-20 at 11 34 01](https://github.com/user-attachments/assets/82a919b3-7d42-414b-8b1c-99ae9caa3bca)